staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -30,7 +29,9 @@ import (
 	"go.etcd.io/etcd/server/v3/embed"
 	"google.golang.org/grpc/grpclog"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Let's not add this - we can create a non-empty object with examplev1, so let's just use that only.

Reply:
fixed